pkg/webui/console/components/device-data-form/index.js

```diff
@@ -67,7 +58,11 @@ class DeviceDataForm extends Component {
 
   @bind
   handleABPSelect() {
-    this.setState({ otaa: false })
+    const { mayEditKeys } = this.props
+
+    if (mayEditKeys) {
```

Comment:
It wouldn't be possible to fire the handler anyway if it is disabled (or the radio button removed).